paasta_tools/mesos_tools.py

```diff
@@ -468,10 +469,15 @@ def get_mesos_state_from_leader():
     return state
 
 
-def get_mesos_quorum(state):
+def get_mesos_state_summary_from_leader():
+    res = master.CURRENT.fetch("/master/state-summary")
```

Comment:
hm, it looks like ``state/summary.json`` *is* exposed on the masters, ``/master/state-summary`` is even less verbose. Infact, ``state/summary.json`` doesn't look like it has any less info than the regular ``state`` endpoint on the master (brief testing of timing curls doesn't show it to be any quicker, either)?

```
robj@robj-yelp-mb:~/workspace/marathon-python fix-travis-py3 % curl -s http://paasta1-uswest1cdevc.dev.yelpcorp.com:5050/state | jq keys
[
  "activated_slaves",
  "build_date",
  "build_time",
  "build_user",
  "cluster",
  "completed_frameworks",
  "deactivated_slaves",
  "elected_time",
  "flags",
  "frameworks",
  "git_sha",
  "git_tag",
  "hostname",
  "id",
  "leader",
  "log_dir",
  "orphan_tasks",
  "pid",
  "slaves",
  "start_time",
  "unregistered_frameworks",
  "version"
]
robj@robj-yelp-mb:~/workspace/marathon-python fix-travis-py3 % curl -s http://paasta1-uswest1cdevc.dev.yelpcorp.com:5050/state/summary.json | jq keys
[
  "activated_slaves",
  "build_date",
  "build_time",
  "build_user",
  "cluster",
  "completed_frameworks",
  "deactivated_slaves",
  "elected_time",
  "flags",
  "frameworks",
  "git_sha",
  "git_tag",
  "hostname",
  "id",
  "leader",
  "log_dir",
  "orphan_tasks",
  "pid",
  "slaves",
  "start_time",
  "unregistered_frameworks",
  "version"
]
robj@robj-yelp-mb:~/workspace/marathon-python fix-travis-py3 % curl -s http://paasta1-uswest1cdevc.dev.yelpcorp.com:5050/state-summary | jq keys
[
  "cluster",
  "frameworks",
  "hostname",
  "slaves"
]
```

There isn't, however, a less verbose option on the slaves. It looks like ``state/summary.json`` is the only 'summary' endpoint available.